reference-implementation/to-upstream-wpts/transform-streams/errors.js

```diff
@@ -232,4 +232,52 @@ promise_test(t => {
   });
 }, 'abort should set the close reason for the writable when it happens first during underlying sink write');
 
+test(() => {
+  new TransformStream({
+    start(controller) {
+      controller.error(thrownError);
+      assert_throws(new TypeError(), () => controller.error(), 'error() should throw');
+    }
+  });
+}, 'controller.error() throws the second time it is called');
+
+promise_test(() => {
+  let controller;
+  const ts = new TransformStream({
+    start(c) {
+      controller = c;
+    }
+  });
+  const cancelPromise = ts.readable.cancel();
+  assert_throws(new TypeError(), () => controller.error(), 'error() should throw');
+  return cancelPromise;
+}, 'controller.error() throws after readable.cancel()');
```

Comment:
"and (but?) the cancel succeeds"? (same for other `return cancelPromise` tests

Reply:
Done.